pygmt/clib/session.py

```diff
@@ -1747,6 +1747,8 @@ def virtualfile_to_dataset(
         vfname: str,
         output_type: Literal["pandas", "numpy", "file"] = "pandas",
         column_names: list[str] | None = None,
```

Comment:
In `Session.virtualfile_to_dataset`, do we want to rename `column_names` to `names`?

Reply:
Done in f211b7d69c4701adf11bf2a0fb4226aa1265c35b.

Reply:
Do you mean using `column_names` in both `GMT_DATASET.to_dataframe` and `Session.virtualfile_to_dataset`?

Reply:
Yes, i.e. revert https://github.com/GenericMappingTools/pygmt/commit/f211b7d69c4701adf11bf2a0fb4226aa1265c35b